Yoast/yoast-components
# Don't encode percent characters that are encoded by node's url.parse

## Summary

This PR can be summarized in the following changelog entry:

* Not applicable

## Relevant technical choices:

* Characters that were previously encoded by `node`'s `url.parse` are now no longer url encoded in the mobile preview's breadcrumbs.

## Test instructions

This PR can be tested by following these steps:

* Create a slug containing some of the following characters (` `^{`(including space)).
* Make sure the characters are not percent encoded in the breadcrumbs preview in mobile mode.

Fixes #514
